### PR TITLE
Add some colour to unit tests

### PR DESCRIPTION
**Pull request by austincunningham**
# What
Add some colours to tests to make it easier to see failing tests, Green for passing test Red for failing e.g.

image

# Verification
- checkout branch
- run `make test` you should see colours in test output
image

### CODE REVIEW

**Comment by eguzki** [comment]
Thanks for the contribution, for unknown reason, tests were not triggered in CircleCI. The "Build forked pull requests" setting is enabled. :shrug: 

I would add in the makefile a dependency and a target to install `gotest` in $PROJECT/bin path avoiding installing globally. 

```
❯ make test
gotest -v github.com/3scale/3scale-porta-go-client/client -test.coverprofile="coverage.txt"
/bin/sh: 1: gotest: not found
make: *** [Makefile:10: test] Error 127
```

**Comment by eguzki** [comment]
Actually, there is no need to include in the go.mod and go.sum

**Comment by austincunningham** [comment]
> I would add in the makefile a dependency and a target to install `gotest` in $PROJECT/bin path avoiding installing globally.

Good idea , yea I suspected circle ci would have a problem with it

File: Makefile [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
+	$(call go-get-tool,$(GOTEST),github.com/rakyll/gotest)
+
 .DEFAULT_GOAL := help
 .PHONY : help
 help: Makefile
 	@sed -n 's/^##//p' $<
 
 ## test: Run unit tests
+.PHONY: test
 test:
```
Comment (eguzki) [inline Makefile:19]: What about adding gotest as dependency? `test: $(GOTEST)`

**Review by eguzki — APPROVED** [review]

**Pull request merged**